io.asgardio.tomcat.saml.agent.sample/src/main/webapp/WEB-INF/classes/META-INF/LICENSE

```diff
@@ -1,200 +1,202 @@
-      by You to the Licensor shall be under the terms and conditions of\n\
-      this License, without any additional terms or conditions.\n\
-      Notwithstanding the above, nothing herein shall supersede or modify\n\
-      the terms of any separate license agreement you may have executed\n\
-      with Licensor regarding such Contributions.\n\
-\n\
-   6. Trademarks. This License does not grant permission to use the trade\n\
-      names, trademarks, service marks, or product names of the Licensor,\n\
-      except as required for reasonable and customary use in describing the\n\
-      origin of the Work and reproducing the content of the NOTICE file.\n\
-\n\
-   7. Disclaimer of Warranty. Unless required by applicable law or\n\
-      agreed to in writing, Licensor provides the Work (and each\n\
-      Contributor provides its Contributions) on an "AS IS" BASIS,\n\
-      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or\n\
-      implied, including, without limitation, any warranties or conditions\n\
-      of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A\n\
-      PARTICULAR PURPOSE. You are solely responsible for determining the\n\
-      appropriateness of using or redistributing the Work and assume any\n\
-      risks associated with Your exercise of permissions under this License.\n\
-\n\
-   8. Limitation of Liability. In no event and under no legal theory,\n\
-      whether in tort (including negligence), contract, or otherwise,\n\
-      unless required by applicable law (such as deliberate and grossly\n\
-      negligent acts) or agreed to in writing, shall any Contributor be\n\
-      liable to You for damages, including any direct, indirect, special,\n\
-      incidental, or consequential damages of any character arising as a\n\
-      result of this License or out of the use or inability to use the\n\
-      Work (including but not limited to damages for loss of goodwill,\n\
-      work stoppage, computer failure or malfunction, or any and all\n\
-      other commercial damages or losses), even if such Contributor\n\
-      has been advised of the possibility of such damages.\n\
-\n\
-   9. Accepting Warranty or Additional Liability. While redistributing\n\
-      the Work or Derivative Works thereof, You may choose to offer,\n\
-      and charge a fee for, acceptance of support, warranty, indemnity,\n\
-      or other liability obligations and/or rights consistent with this\n\
-      License. However, in accepting such obligations, You may act only\n\
-      on Your own behalf and on Your sole responsibility, not on behalf\n\
-      of any other Contributor, and only if You agree to indemnify,\n\
-      defend, and hold each Contributor harmless for any liability\n\
-      incurred by, or claims asserted against, such Contributor by reason\n\
-      of your accepting any such warranty or additional liability.\n\
-\n\
-   END OF TERMS AND CONDITIONS\n\
+
```

Comment:
Do we need this?

Reply:
Removed the file in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/2/commits/7bc0a7b510c5137c156f5317497b12e794ef3993